<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>485</width>
    <height>485</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PaletteMaker</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
background-color: rgb(50, 50, 50);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget{
background-color: rgb(50, 50, 50);
}</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="source_label">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel{
	color: rgb(255, 255, 255);
	font: bold 15px;
	border-radius: 6px;
    padding: 2px;
}</string>
          </property>
          <property name="text">
           <string>Source Image</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="source_selection_cb">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="focusPolicy">
           <enum>Qt::WheelFocus</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">QComboBox {
    border: 1px solid gray;
    border-radius: 3px;
    padding: 3px 18px 3px 3px;
    min-width: 6em;
	color: rgb(50, 50, 50);
	font: bold 15px;
}

QComboBox:!editable, QComboBox::drop-down:editable {
     background: white
}

QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 25px;

}

QListView{
	background: white
}

</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="palette_label">
          <property name="styleSheet">
           <string notr="true">QLabel{
	color: rgb(255, 255, 255);
	font: bold 15px;
	border-radius: 6px;
    padding: 2px;
}</string>
          </property>
          <property name="text">
           <string>Palette Name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="palette_name_le">
          <property name="styleSheet">
           <string notr="true">QLineEdit{
background-color: rgb(255, 255, 255);
color: rgb(50, 50, 50);
font: bold 15px;;
}</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="generate_btn">
        <property name="styleSheet">
         <string notr="true">QPushButton{
	background-color:  rgb(255, 255, 255);
	font: bold 15px;
	color: rgb(50, 50, 50)
}
QPushButton:hover{
	background-color: rgb(255, 143, 197);
	font: bold 15px;
	color:   rgb(50, 50, 50)
}
QPushButton:pressed{
	background-color:rgb(250, 160, 197);
	font: bold 15px;
	color:  rgb(50, 50, 50)
}</string>
        </property>
        <property name="text">
         <string>Generate Palette</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="result_text_label">
            <property name="styleSheet">
             <string notr="true">QLabel{
	color: rgb(255, 255, 255);
	font: bold 15px;
	border-radius: 6px;
    padding: 2px;
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="srcimg_label">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="srcimsg_label">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="result_label">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLineEdit" name="colour_values_le">
            <property name="styleSheet">
             <string notr="true">QLineEdit{
background-color: rgb(255, 255, 255);
color: rgb(50, 50, 50);
}</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
